### PR TITLE
Clarified Windows development installation and docd upstream enchant bug

### PR DESCRIPTION
**Pull request by Badg**
This is a proposed docs clarification to address #3127

### CODE REVIEW

**Comment by alex** [comment]
Jenkins, ok to test

**Comment by alex** [comment]
This PR appears to be based on an older master, and thus there are merge conflicts. You should merge/rebase master.

**Comment by Badg** [comment]
Merged. Conflict was minor, just missing 

> * ``OpenSSL 1.1.0``

for openssl support.

File: docs/installation.rst [hunk 1]
```diff
@@ -38,13 +38,14 @@ OpenSSL releases:
 
 .. warning::
     OpenSSL 1.0.0 is no longer supported by the OpenSSL project. Cryptography
-    will drop support for it in the next release.
+    will drop support for it in a future release.
```
Comment (reaperhulk) [inline docs/installation.rst]: This line needs to be changed back to `the next` -- looks like a merge resolution mistake.

Comment (Badg) [inline docs/installation.rst]: Good catch!

**Pull request merged**